### DISCUSSION TITLE
[messaging] Is there a way to disabled headless task in android & ios?

### DISCUSSION THREAD

**Discussion by kesha-antonov** [invertase/react-native-firebase#7725] [Q&A]
Hi

By default firebase sets headless task https://github.com/invertase/react-native-firebase/blob/2cee82424b7191c83d5378209a31cb73cbf8bd0a/packages/messaging/lib/index.js#L112

But in our app this feature is not needed. We don't need to start app when it's in the killed state.

Is there a way to disable this feature in ios + android?

**Comment by ilyagru** [comment]
Another question: Is it possible to start the app as usual instead of the headless task on notification?
